ietf/doc/views_search.py

```diff
@@ -345,44 +345,35 @@ def ad_dashboard_group(doc):
         if doc.get_state_slug('draft') == 'rfc':
             return 'RFC'
         elif doc.get_state_slug('draft') == 'active' and doc.get_state_slug('draft-iesg'):
-            return '%s Internet-Draft' % doc.get_state('draft-iesg').name
+            return str(doc.get_state('draft-iesg').name)
         else:
-            return '%s Internet-Draft' % doc.get_state('draft').name
+            return str(doc.get_state('draft').name)
     elif doc.type.slug=='conflrev':
         if doc.get_state_slug('conflrev') in ('appr-reqnopub-sent','appr-noprob-sent'):
-            return 'Approved Conflict Review'
+            return 'Approved'
         elif doc.get_state_slug('conflrev') in ('appr-reqnopub-pend','appr-noprob-pend','appr-reqnopub-pr','appr-noprob-pr'):
-            return "%s Conflict Review" % State.objects.get(type__slug='draft-iesg',slug='approved')
+            return str(State.objects.get(type__slug='draft-iesg',slug='approved'))
         else:
-          return '%s Conflict Review' % doc.get_state('conflrev')
+          return str(doc.get_state('conflrev'))
     elif doc.type.slug=='statchg':
         if doc.get_state_slug('statchg') in ('appr-sent',):
-            return 'Approved Status Change'
+            return 'Approved'
         if doc.get_state_slug('statchg') in ('appr-pend','appr-pr'):
-            return '%s Status Change' % State.objects.get(type__slug='draft-iesg',slug='approved')
+            return str(State.objects.get(type__slug='draft-iesg',slug='approved'))
         else:
-            return '%s Status Change' % doc.get_state('statchg')
+            return str(doc.get_state('statchg'))
     elif doc.type.slug=='charter':
         if doc.get_state_slug('charter') == 'approved':
-            return "Approved Charter"
+            return "Approved"
```

Comment:
Not sure why this is done differently than just getting the state name?
```
>>> State.objects.get(type='charter', slug='approved').name
'Approved'
```

Reply:
Don't know either, I just kept the logic from before